### PR TITLE
Kinetics and mopac unittests

### PR DESCRIPTION
**Pull request by mjohnson541**
Some basic unit-tests for data/kinetics and mopac

the changes to mopac.py cause mopac to verify its output files after its test runs

in data/kinetics/kineticsTest.py

* testFilterReactions - tests that filterReactions output includes all of the reactions it should

* testSaveEntry - tests that running saveEntry on reactions does not produce an error

* testDuplicates - tests that kinetic libraries load properly and that the duplicate management functions do not give false positives

### CODE REVIEW

**Comment by KEHANG** [comment]
@mjohnson541 before merging, please refer PR #953 on how to load RMG database in unittest

**Comment by mjohnson541** [comment]
I'm not very familiar with the side effects of the decorators used in #953 on the nosetests, I'm getting problems when I add the tearDownClass function, it seems to interfere with the earlier tests and cause them to fail.

**Comment by mliu49** [comment]
Could rebase this branch on top of `fixDatabaseTests` and see if the tests pass then?

**Comment by mjohnson541** [comment]
Yep, that makes it work!

**Comment by KEHANG** [comment]
@mjohnson541 #953 is to be merged soon. You might want to rebase this branch again and it also needs change a little bit since `kineticsTest.py` now loads database once for the entire module

**Comment by mjohnson541** [comment]
Those changes are made and this should be good to push now.

**Comment by mliu49** [comment]
Could you remove 0fe7b84 and then combine all of the commits adding tests for kinetics.common?

File: rmgpy/data/kinetics/kineticsTest.py [hunk 1]
```diff
@@ -509,4 +510,94 @@ def testParseKinetics(self):
...
         self.assertEqual(sources[4]['PDep'], 7)
+
+
+class TestKinetics(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(self):
+        from rmgpy.chemkin import loadChemkinFile
+        """
+        A function run before each unit test in this class.
```
Comment (mliu49) [inline rmgpy/data/kinetics/kineticsTest.py]: Could you update this docstring?

File: rmgpy/data/kinetics/kineticsTest.py [hunk 2]
```diff
@@ -509,4 +510,94 @@ def testParseKinetics(self):
...
+        self.assertIsNone(out)
+        out = lib.convertDuplicatesToMulti()
+        self.assertIsNone(out)
+        
+        
+        
+        
+        
+        
+
```
Comment (mliu49) [inline rmgpy/data/kinetics/kineticsTest.py]: Remove extra blank lines?

File: rmgpy/qm/mopacTest.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 import shutil
 import subprocess
 
+from rmgpy import settings
```
Comment (mliu49) [inline rmgpy/qm/mopacTest.py]: I think this is an unused import?

Comment (mjohnson541) [inline rmgpy/qm/mopacTest.py]: You need it to get the proper directory paths.

Comment (mjohnson541) [inline rmgpy/qm/mopacTest.py]: Oh, sorry, looking at the wrong change

**Comment by mjohnson541** [comment]
Ok, those changes are made.

**Pull request merged**